.vscode/launch.json

```diff
@@ -35,6 +35,20 @@
           "VSC_DEBUG": "true"
         }
       }
+    },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "Jest Current File (webview)",
+      "program": "${workspaceFolder}/node_modules/.bin/jest",
+      "args": ["${fileBasenameNoExtension}", "--config", "jest.config.js"],
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "disableOptimisticBPs": true,
+      "windows": {
+        "program": "${workspaceFolder}/node_modules/jest/bin/jest"
+      },
+      "cwd": "${workspaceFolder}/webview"
```

Comment:
[I] It would be better to add this as a task to tasks.json. Something like:

```
    {
      "args": [
        "workspace",
        "dvc-vscode-webview",
        "jest",
        "${fileBasenameNoExtension}"
      ],
      "type": "process",
      "isBackground": false,
      "label": "Jest Current File (webview)",
      "command": "yarn",
      "options": { "cwd": "${workspaceFolder}" }
    }

```

Reply:
Generally we run these commands in our terminals.

Reply:
From running the tests it appears that this change has caused a lot of noise in the logs:

https://github.com/iterative/vscode-dvc/actions/runs/3611484435/jobs/6085998133#step:8:697

Reply:
> It would be better to add this as a task to tasks.json

any specific reason for this?

> From running the tests it appears that this change has caused a lot of noise in the logs:

looking into this ... I skipped `ul` consciously. To avoid an empty invisible block when there are no items. I'll look how to silent of skip `ul` in a different way

Reply:
> any specific reason for this?

Yes, the task API is generally what you would use for running dev processes in a more "VS Code way":

```
Tasks in VS Code can be configured to run scripts and start processes so that many of these existing tools can be used from within VS Code without having to enter a command line or write new code. Workspace or folder specific tasks are configured from the tasks.json file in the .vscode folder for a workspace.

```
Docs: https://code.visualstudio.com/docs/editor/tasks

If you are actually using breakpoints with the tests. Please disregard.

Reply:
> If you are actually using breakpoints with the tests. Please disregard.

Yep, that was the only reason to add this. I'm trying to run pretty everything under debugger - makes it easier to see things right away.